File: 30-days-of-code/java/Day2.java
import java.util.*;
import java.math.*;

public class Day2 {

public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble(); // original meal price
        int tipPercent = scan.nextInt(); // tip percentage
        int taxPercent = scan.nextInt(); // tax percentage
        scan.close();

        double percentBase = 100.0;
        double tip = tipPercent / percentBase;
        double tax = taxPercent / percentBase;

        // cast the result of the rounding operation to an int and save it as totalCost
        int totalCost = (int) Math.round(mealCost + mealCost * tip + mealCost * tax);

        System.out.println("The total meal cost is " + totalCost + " dollars.");
}
}

File: 30-days-of-code/go/Day2.go
package main

import (
	"fmt"
	"math"
)

func round(f float64) uint64 {
	return uint64(math.Floor(f + .5))
}

func main() {
	var (
		mealCost   float64
		tipPercent float64
		taxPercent float64
	)

	fmt.Scan(&mealCost)
	fmt.Scan(&tipPercent)
	fmt.Scan(&taxPercent)

	tipCost := tipPercent / 100 * mealCost
	taxCost := taxPercent / 100 * mealCost
	totalCost := tipCost + taxCost + mealCost

	fmt.Printf("The total meal cost is %d dollars.", round(totalCost))

}
